ci/setup-cluster-api.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source "$(dirname "$0")/common-env"
+
+# This script expects the following env vars to be set
+#  OCI_USER_ID
+#  OCI_CREDENTIALS_FINGERPRINT
+#  OCI_CREDENTIALS_KEY
```

Comment:
Worth some basic dumb bash to exit early if these aren't set?

Reply:
Probably. I'll add that in